src/screens/BenefitManipulationScreen.tsx

```diff
@@ -108,7 +111,7 @@ export default function BenefitManipulationScreen({ navigation, Benefit }) {
               name='benefits'
               isInvalid={Boolean(errors.benefits)}
             />
```

Comment:
error zjada komponent + brak tekstu

![image](https://github.com/StampWallet/stamp-wallet/assets/93001484/b41908b6-435b-4246-a231-c32c7408b97f)

Reply:
Który error zjada komponent?

Reply:
error message ma szerokosc 100% przez co error w lewym datepickerze przyslanial prawy datepicker, dorzucilem kontenery zeby szerokosc sie do nich odnosila, na razie wyglada to tak
![image](https://github.com/StampWallet/stamp-wallet/assets/93001484/85bb4216-18cc-4af7-a80a-2458d9c1a22f)

Reply:
Jest good!